ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -134,6 +134,7 @@ public class InvoiceDescriptor
         public List<TradeAllowanceCharge> TradeAllowanceCharges { get; set; } = new List<TradeAllowanceCharge>();
         public PaymentTerms PaymentTerms { get; set; }        
         public InvoiceReferencedDocument InvoiceReferencedDocument { get; set; }
+        public List<ReceivableSpecifiedTradeAccountingAccount> ReceivableSpecifiedTradeAccountingAccounts { get; set; } = new List<ReceivableSpecifiedTradeAccountingAccount>();
```

Comment:
could you give the setter internal visibility so that users make use of the Add() functions instead of adding directly to the list?

Reply:
I added "internal"...